### PR TITLE
AETHER-2874 Bootstrap aether-2.x models

### PR DESCRIPTION
**Pull request by sbconsulting**

### CODE REVIEW

File: modelplugin/aether-2.0.0/modelplugin/modelplugin.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
+// Copyright 2021-present Open Networking Foundation.
```
Comment (SeanCondon) [inline modelplugin/aether-2.0.0/modelplugin/modelplugin.go:1]: I think this file is not needed - but I was expecting to see a `generated.go`

**Review by SeanCondon — APPROVED** [review]

File: modelplugin/aether-2.0.0/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# YANG files
+The YANG files to accompany this plugin have been moved to 
+
+* ../../../sdran-helm-charts/config-models/aether-3.x/files/yang
```
Comment (kmarquardsen) [inline modelplugin/aether-2.0.0/README.md:4]: isn't this ../../..//roc-helm-charts/config-models/aether-2.0.x/files/yang ?

Comment (sbconsulting) [reply modelplugin/aether-2.0.0/README.md:4]: yeah, I'll update the readme.

File: modelplugin/aether-2.0.0/modelplugin/modelplugin.go (hunk 1 above)
Comment (sbconsulting) [reply modelplugin/aether-2.0.0/modelplugin/modelplugin.go:1]: probably forgot to add it. Let me see if sdcore-adapter needs the modelplugin.go.

File: modelplugin/aether-2.0.0/generator.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+mkdir -p aether_2_0_0
+
+OPENCORD="${OPENCORD:-$HOME/git/opencord}"
+
+go run github.com/openconfig/ygot/generator -path=$OPENCORD/roc-helm-charts//config-models/aether-4.x/files/yang \
```
[the remark this answers is not in the document]
Comment (kmarquardsen) [reply modelplugin/aether-2.0.0/generator.sh]: Also should be 2.0.x now?

**Review by kmarquardsen — APPROVED** [review]

File: modelplugin/aether-2.0.0/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# YANG files
```
Comment (kmarquardsen) [inline modelplugin/aether-2.0.0/README.md:1]: could just delete this file since the yang files used to generate are "documented" in generator.sh I guess

**Pull request merged**